labs/devnet-alexa-lab/2.md

```diff
@@ -24,17 +24,14 @@ Follow these steps to set up your environment and get you ready to put together
 
    and press enter.
 
-4. Get the code with a Git command, where this code becomes the **Lambda Function**.
-   Type this command: `git clone https://github.com/movinalot/devnet-alexa.git` and press enter.
+4. Get the code with a Git command, where this code becomes the **Lambda Function**. Type this command: `git clone https://github.com/movinalot/devnet-alexa.git` and press enter. *Note: It may take a few moments for the code to download*.
 
-    **It will take a few moments for the code to download**.
-
-  You just **cloned** (copied) code to your computer from **github.com** This is the code that runs in AWS in the Lambda service when your voice request triggers it.
+  You just **cloned** (copied) code to your computer from **github.com**. This is the code that runs in AWS in the Lambda service when your voice request triggers it.
 
   **Get the Skill Code**
   ![](assets/images/image-09.jpg)<br/><br/>
 
-#### Make and Activate a Virtual Environment
+## Make and Activate a Virtual Environment
```

Comment:
Lowercase activate for sentence-style capitalization. May want to do the same for virtual environment since we don't have to treat that as a proper noun.